site/content/docs/user/known-issues.md

```diff
@@ -248,6 +249,17 @@ open /home/user/.docker/config.json: permission denied
 To fix this problem, either follow the docker's docs [manage docker as a non root user][manage docker as a non root user],
 or try to use `sudo` before your commands (if you get `command not found` please check [this comment about sudo with kind][sudo with kind]).
 
+
+## Docker on Windows
+
+[Docker Desktop for Windows][docker desktop for windows] supports running both Linux (the default) and Windows Docker containers.
+
+`kind` will fail create a cluster, if your Docker Desktop for Windows using `Windows containers` mode.
+
+This is because `kind` only distributed docker images base on Linux.
+
+To fix this problem, you can follow the docker's docs [switch between Windows and Linux containers][switch between windows and linux containers].
```

Comment:
this is by design. should this really be in "known issues"?

i think the whole section can be summarized to:

```
## Docker on Windows

[Docker Desktop for Windows][docker-desktop] supports running both Linux (the default) and Windows Docker containers.

kind for Windows requires Linux containers. To switch between Linux and Windows containers see [this page][docker-desktop-switch].
```

Reply:
I think that summary is perfect!

Reply:
Great!

Reply:
+1

Reply:
I think this summary is clear :+1:  
I want to make changes based on this suggestions.